sql-tests/tests/snapshots/tests__testdata__create_ingest_temp_table.sql.snap

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Why are these files not registered as a "move"? Do we also need to remove them from `e2e`?

Reply:
Thank you!